sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -6,40 +6,46 @@
 # pylint: disable=docstring-keyword-should-match-keyword-only, too-many-lines
 
 from typing import (
-    Any, Dict, Iterable, Literal, Optional, Union,
+    Any, cast, Dict, Literal, Optional, Union,
     TYPE_CHECKING
 )
-from typing_extensions import Self
+from typing_extensions import Never, Self
 
 from azure.core.exceptions import HttpResponseError
-from azure.core.tracing.decorator import distributed_trace
+from azure.core.paging import ItemPaged
 from azure.core.pipeline import Pipeline
+from azure.core.tracing.decorator import distributed_trace
 from ._deserialize import deserialize_permission, deserialize_share_properties
 from ._directory_client import ShareDirectoryClient
 from ._file_client import ShareFileClient
 from ._generated import AzureFileStorage
 from ._generated.models import (
     DeleteSnapshotsOptionType,
-    SignedIdentifier)
+    ShareStats,
+    SignedIdentifier
+)
 from ._lease import ShareLeaseClient
 from ._models import ShareProtocols
+from ._parser import _parse_snapshot
 from ._serialize import get_api_version, get_access_conditions
 from ._share_client_helpers import (
     _create_permission_for_share_options,
     _format_url,
     _from_share_url,
-    _parse_url)
+    _parse_url
+)
 from ._shared.base_client import StorageAccountHostsMixin, TransportWrapper, parse_connection_str, parse_query
 from ._shared.request_handlers import add_metadata_headers, serialize_iso
 from ._shared.response_handlers import (
     return_response_headers,
     process_storage_error,
-    return_headers_and_deserialized)
+    return_headers_and_deserialized
+)
```

Comment:
Same thing here, p > r

Overall, I would just take this comment to sweep through the imports one time for both files in this PR 😄

Reply:
Swept thru one more time, should be good now 😁